### ISSUE TITLE
add manifest declaration for Win10 DPI support

### ISSUE THREAD

**Issue by sean-e** [sean-e/ClipWatch#5]
https://blogs.windows.com/buildingapps/2017/05/19/improving-high-dpi-experience-gdi-based-desktop-apps/
```xml
<asmv3:application>
	<asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2017/WindowsSettings">
		<gdiScaling>true</gdiScaling>
	</asmv3:windowsSettings>
</asmv3:application>
```

**Comment by sean-e** [comment]
implemented in 3cde783b1ecc6ef4b991afabace6fd74b32c9c12

**Comment by sean-e** [comment]
https://msdn.microsoft.com/en-us/library/windows/desktop/aa374191%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396
